src/dbnode/ts/types.go

```diff
@@ -72,9 +76,13 @@ type Series struct {
 	// ID is the series identifier.
 	ID ident.ID
 
-	// Tags are the series tags.
+	// Tags is the series tags.
 	Tags ident.Tags
```

Comment:
Is this still used anywhere?

Reply:
Yeah, it's used on the read side and it was going to be very difficult to remove unfortunately.